examples/reference/elements/matplotlib/QuadMesh.ipynb

```diff
@@ -88,6 +88,40 @@
     "\n",
     "For full documentation and the available style and plot options, use ``hv.help(hv.QuadMesh).``"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Irregular meshes\n",
+    "\n",
+    "In addition to axis aligned meshes like those we worked with above, a ``QuadMesh`` may also be used to represent irregular meshes. In this example we will create an irregular mesh consisting of 2D X, Y and Z arrays defining the position and value of each simplex in the mesh:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "n=20\n",
+    "coords = np.linspace(-1.5,1.5,n)\n",
+    "X,Y = np.meshgrid(coords, coords);\n",
+    "Qx = np.cos(Y) - np.cos(X)\n",
+    "Qz = np.sin(Y) + np.sin(X)\n",
+    "Z = np.sqrt(X**2 + Y**2)\n",
+    "\n",
+    "print(Qx.shape, Qz.shape, Z.shape)\n",
```

Comment:
Stray print? If not it needs to print a proper message.

Reply:
Sure, just meant to show how the data differs between unevenly sampled and irregularly sampled QuadMeshes.

Reply:
Ok. Then I would draw attention to that more explicitly including a proper print message, maybe in a separate cell.